pkgs/applications/misc/crow-translate/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv
+, mkDerivation
+, fetchFromGitHub
+, substituteAll
+, cmake
+, extra-cmake-modules
+, qttools
+, leptonica
+, tesseract4
+, qtmultimedia
+, qtx11extras
+, qttranslations
+}:
+let
+  singleapplication = fetchFromGitHub {
+    owner = "itay-grudev";
+    repo = "SingleApplication";
+    rev = "f3de0114f45c7ff073f29cf0822df5161bb5c3b3";
+    sha256 = "06as0klwy3gckf9j42nx5ybfvxmqc3qkd92yvsgvax3x7v57m62w";
+  };
+  qtaskbarcontrol = fetchFromGitHub {
+    owner = "Skycoder42";
+    repo = "QTaskbarControl";
+    rev = "2.0.2";
+    sha256 = "0iymcvq3pv07fs9l4kh6hi1igqr7957iqndhsmg9fqkalf8nqyad";
+  };
+  qhotkey = fetchFromGitHub {
+    owner = "Skycoder42";
+    repo = "QHotkey";
+    rev = "1.4.2";
+    sha256 = "0391fkqrxqmzpvms4rk06aq05l308k6sadp6y3czq0gx2kng8mn9";
+  };
+  qonlinetranslator = fetchFromGitHub {
+    owner = "crow-translate";
+    repo = "QOnlineTranslator";
+    rev = "1.4.0";
+    sha256 = "045ah3gw05k8md0hgf79npcj4g3acinx2ak9hqyfql722ksxx50l";
+  };
+in
+mkDerivation rec {
+  pname = "crow-translate";
+  version = "2.6.1";
+
+  src = fetchFromGitHub {
+    owner = "crow-translate";
+    repo = "crow-translate";
+    rev = version;
+    sha256 = "04bmdfcjnw0gm64r4fnqyvvsf1i9gl9212i80bvbn0d238bpyang";
+  };
+
+  patches = [
+    (substituteAll {
+      src = ./dont-fetch-external-libs.patch;
+      inherit singleapplication qtaskbarcontrol qhotkey qonlinetranslator;
```

Comment:
Could those libs be used in other packages? If so please create one package for each of them. This can be done in this PR but in different commits.

Reply:
Unlikely, moreover, it doesn't make much sense, since it's just c++ sources library, that is nothing more than fetchFromGitHub. Secondly, the upstream clearly pins those libraries to specific rev/tag.

Reply:
> the upstream clearly pins those libraries to specific rev/tag.

Generally we take this a bit relaxed.